src/abe_policy/policy.rs

```diff
@@ -325,7 +313,7 @@ impl Policy {
         };
 
         let current_axis_values = attr_values_per_axis.get(current_axis_name).ok_or_else(|| {
-            Error::Other(format!(
+            Error::InvalidAxis(format!(
                 "no attribute value found for axis: {current_axis_name}",
             ))
```

Comment:
`InvalidAxis` should accept only the `current_axis_name` and not the full description as string

Reply:
ok

Reply:
done in 3a2eb71